ansible_collections/arista/avd/docs/contribution/style-guide.md

```diff
@@ -38,10 +38,10 @@ When variables are used in combination with a filter, `|` shall be enclosed by s
 
 Nested jinja code block shall follow next rules:
 
-* All J2 statements must be enclosed by 1 space
-* All J2 statements must be indented by 4 more spaces within jinja delimiter
-* To close a control, end tag must have same indentation level
-* Indentation are 4 spaces and NOT tabulation
+- All J2 statements must be enclosed by 1 space
+- All J2 statements must be indented by 4 more spaces within jinja delimiter
+- To close a control, end tag must have same indentation level
+- Indentation should be 4 spaces and NOT tabulation
```

Comment:
must be, not should

Reply:
Fixed. Also, I fixed the grammar(used grammarly) in this page.